packages/@vuepress/core/lib/client/root-mixins/updateMeta.js

```diff
@@ -35,10 +35,26 @@ export default {
 
     getMergedMetaTags () {
       const pageMeta = this.$page.frontmatter.meta || []
+      const templateMeta = []
+      if (typeof document !== 'undefined') {
+        const templateMetaTags = document.querySelectorAll('meta')
```

Comment:
`this.currentMetaTag` already gather meta tag on mount and reusing may provide better code readability

Reply:
@NeverBehave thanks!